lib/settings_base.py

```diff
@@ -1247,15 +1247,21 @@ def JINJA_CONFIG():
 CSP_REPORT_ONLY = True
 
 CSP_DEFAULT_SRC = ("'self'",)
-CSP_SCRIPT_SRC = (
+CSP_IFRAME_SRC = (
+    "'self'",
+    'https://*.paypal.com',
```

Comment:
Does that mean that by default on prod we're allowing everything, and then only restricting to `www.sandbox....` on -dev and stage? Maybe it's missing an explicit `https://www.paypal.com` on prod?